experimental/incremental/queries/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// package queries provides specific [incremental.Query]s for various parts of
```

Comment:
"querys" really doesn't look right 😝 
```suggestion
// package queries provides specific [incremental.Query] types for various parts of
```